HEAD no longer builds

Building from HEAD produces this error:

```
./uncrustify/src/brace_cleanup.cpp:424:10: error: 
      no matching function for call to 'close_statement'
         close_statement(frm, pc);
         ^~~~~~~~~~~~~~~
./uncrustify/src/brace_cleanup.cpp:67:13: note: 
      candidate function not viable: requires 3 arguments, but 2 were provided
static bool close_statement(ParseFrame &frm, chunk_t *pc, const BraceSta...
            ^
1 error generated.

```

--- a/src/brace_cleanup.cpp
+++ b/src/brace_cleanup.cpp
@@ -421,7 +421,7 @@ static void parse_cleanup(BraceState &braceState, ParseFrame &frm, chunk_t *pc)
       else if (  language_is_set(LANG_D)
               && chunk_is_token(pc, CT_BRACE_CLOSE))
       {
-         close_statement(frm, pc);
+         close_statement(frm, pc, braceState);
       }
    }
 

